.github/actions/javascript/createOrUpdateStagingDeploy/index.js

```diff
@@ -582,11 +592,13 @@ class GithubUtils {
                     });
                 }
 
+                // Internal QA PR list
                 if (!_.isEmpty(internalQAPRMap)) {
+                    console.log('Found the following verified Internal QA PRs:', resolvedInternalQAPRs);
                     issueBody += '\r\n\r\n\r\n**Internal QA:**';
                     _.each(internalQAPRMap, (assignees, URL) => {
                         const assigneeMentions = _.reduce(assignees, (memo, assignee) => `${memo} @${assignee}`, '');
-                        issueBody += `\r\n${_.contains(verifiedOrNoQAPRs, URL) ? '- [x]' : '- [ ]'} `;
+                        issueBody += `\r\n${_.contains(resolvedInternalQAPRs, URL) ? '- [x]' : '- [ ]'} `;
```

Comment:
Why is there so much code duplication for creating the internal QA PR list and `getStagingDeployCashData` in these Github actions? I'm very new to Github actions.

Reply:
Check out the documentation [here](https://github.com/Expensify/App/blob/main/.github/workflows/README.md#actions)